client/blocks/image-editor/image-editor-canvas.jsx

```diff
@@ -1,10 +1,11 @@
+/*eslint-disable*/
```

Comment:
Did you mean to add this? If there are eslint errors, we can spin up another janitorial to fix those outside of the PR first.